src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Common.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="12.0" DefaultTargets="PublishOutputLeg" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <RunningOnCore Condition="'$(MSBuildRuntimeType)' == 'Core'">true</RunningOnCore>
+    <_TaskDir>$(MSBuildThisFileDirectory)net46/</_TaskDir>
```

Comment:
I don't think these paths are going to work for both of our scenarios.

Reply:
This seems to work based on testing